app/_src/policies/meshhttproute.md

```diff
@@ -302,6 +304,52 @@ spec:
 {% endtab %}
 {% endtabs %}
 
+{% endif_version %}
+
+{% if_version gte:2.9.x %}
+
+### Traffic split
+
+We can use `MeshHTTPRoute` to split an HTTP traffic between different MeshServices
+implementing A/B testing or canary deployments. 
+If we want to split traffic between `v1` and `v2` versions of the same service,
+first we have to create MeshServices `backend-v1` and `backend-v2` that select 
```

Comment:
do we have docs on how to manually create MeshService? Can we link it here?

Reply:
AFAIK there is only https://kuma.io/docs/2.9.x/networking/meshservice/, do you think we should link it?

Reply:
created an issue https://github.com/kumahq/kuma-website/issues/2028